mist-lib/src/services/stp-utils.js

```diff
@@ -0,0 +1,36 @@
+'use strict'
+
+const csv = require('csvtojson')
+
+exports.parseCSVTable = (tableFilePath) => {
```

Comment:
`parseCSVTable` is fairly generic and does not capture the intent of what this method does (produce a set of pfql rules). Recommend renaming to `readPFQLRulesFromCSVFile`.